_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,263 @@
+---
+id: b8bb68
+name: Text spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+  This rule checks that the text spacing styles can be adjusted by user, i.e. that `!important` is not applied to the style, unless the style has already exceeded the specified metric for retaining content visibility and functionality.
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][], for which one of the following is true:
+
+- the `style` attribute [declares][declared][][word-spacing][] CSS property; or
+- the `style` attribute [declares][declared][][letter-spacing][] CSS property; or
+- the `style` attribute [declares][declared][][line-height][] CSS property.
```

Comment:
Think we concluded to split this rule, right?

Reply:
Yes that is right & here is the secondary PR, where there are 3 atomic rules addressing test spacing.